docs/build/EVM/precompiles/staking.md

```diff
@@ -1,5 +1,6 @@
-# DApp Staking
+# dApp Staking
 
-The dApp Staking Precompile allows EVM smart contracts to access `pallet-dapps-staking` functionality.
+The dApp Staking Precompile allows EVM smart contracts to access `pallet-dapp-staking-v3` functionality.
```

Comment:
I'd drop out the `v3`. In a month, we'll rename the pallet and remove `v3` notation.